lib/ui/semantics.dart

```diff
@@ -181,29 +148,62 @@ class SemanticsAction {
   /// (with TalkBack) special hint text is spoken when focusing the node and
   /// a custom action is available in the local context menu. On iOS,
   /// (with VoiceOver) users can perform a standard gesture to dismiss it.
-  static const SemanticsAction dismiss = SemanticsAction._(_kDismissIndex);
+  dismiss(_kDismissIndex),
 
   /// Move the cursor forward by one word.
   ///
   /// This is for example used by the cursor control in text fields.
   ///
   /// The action includes a boolean argument, which indicates whether the cursor
   /// movement should extend (or start) a selection.
-  static const SemanticsAction moveCursorForwardByWord = SemanticsAction._(_kMoveCursorForwardByWordIndex);
+  moveCursorForwardByWord(_kMoveCursorForwardByWordIndex),
 
   /// Move the cursor backward by one word.
   ///
   /// This is for example used by the cursor control in text fields.
   ///
   /// The action includes a boolean argument, which indicates whether the cursor
   /// movement should extend (or start) a selection.
-  static const SemanticsAction moveCursorBackwardByWord = SemanticsAction._(_kMoveCursorBackwardByWordIndex);
+  moveCursorBackwardByWord(_kMoveCursorBackwardByWordIndex);
+
+  const SemanticsAction(this.value);
+
+  /// The numerical value for this action.
+  ///
+  /// Each action has one bit set in this bit field.
+  final int value;
```

Comment:
Is there a good reason to change the name of this property from `index` to `value`?

Reply:
enum has its own index, so you can't have two index with the same name

Reply:
Oh, right! Makes sense.